app/code/Magento/InventoryCatalog/Observer/ProcessSourceItemsObserver.php

```diff
@@ -73,6 +73,9 @@ public function execute(EventObserver $observer)
     {
         /** @var ProductInterface $product */
         $product = $observer->getEvent()->getProduct();
+        if ($product->getTypeId() == 'bundle') {
```

Comment:
Need to introduce new module `InventoryBundle`
And disable this observer in new module
Test also need to move to new module

Reply:
Also need to use this constant
`\Magento\Bundle\Model\Product\Type::TYPE_CODE`

Reply:
And looks like we need to disable data provider plugin in Inventory module

Reply:
Valeriy, I don't get which plugin in Inventory module should be disabled. There is not any plugins.